openml/tasks/split.py

```diff
@@ -108,7 +110,11 @@ def _from_arff_file(cls, filename: str) -> 'OpenMLSplit':
                     repetitions[repetition][fold][sample] = ([], [])
                 split = repetitions[repetition][fold][sample]
 
-                type_ = line[type_idx].decode('utf-8')
+                if not isinstance(line[type_idx], str):
```

Comment:
Is this case actually triggered with liac-arff?

Reply:
Hi,
The decode was required for how the file was being read earlier. With liac-arff it reads as str. So yes you are correct. This if-statement is not really triggered.
Removing it.